css/css-fonts/variations/font-style-parsing.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <title>Testing the new font-style values introduced in CSS Fonts level 4</title>
+    <link rel="help" href="https://www.w3.org/TR/css-fonts-4/#font-style-prop" />
+    <script src="/resources/testharness.js"></script>
+    <script src="/resources/testharnessreport.js"></script>
+</head>
+<body>
+
+    <script>
+
+        var testFontStyle = [
+            { style: "italic",            expectedResult: true,  message: "'italic' is valid" },
+            { style: "italic 20deg",      expectedResult: false, message: "'italic' followed by angle is invalid" },
+            { style: "italic a",          expectedResult: false, message: "'italic' followed by non-number is invalid" },
+            { style: "oblique",           expectedResult: true,  message: "'oblique' is valid" },
+            { style: "oblique 0deg",      expectedResult: true,  message: "'oblique' followed by zero degrees is valid" },
+            { style: "oblique 20deg",     expectedResult: true,  message: "'oblique' followed by positive angle in degrees is valid" },
+            { style: "oblique 0.5rad",    expectedResult: true,  message: "'oblique' followed by positive angle in radians is valid" },
+            { style: "oblique 20grad",    expectedResult: true,  message: "'oblique' followed by positive angle in gradians is valid" },
+            { style: "oblique 0.1turn",   expectedResult: true,  message: "'oblique' followed by positive angle in turns is valid" },
+            { style: "oblique 20px",      expectedResult: false, message: "'oblique' followed by number with invalid unit type is in valid" },
+            { style: "oblique -20deg",    expectedResult: true,  message: "'oblique' followed by negative angle is valid" },
+            { style: "oblique 20.1deg",   expectedResult: true,  message: "'oblique' followed by fractional angle is valid" },
+            { style: "oblique 90deg",     expectedResult: true,  message: "'oblique' followed by maxumum 90 degree angle is valid" },
+            { style: "oblique -90deg",    expectedResult: true,  message: "'oblique' followed by minimum -90 degree angle is valid" },
+            { style: "oblique 90.01deg",  expectedResult: false, message: "'oblique' followed by positive out of range angle is in invalid" },
+            { style: "oblique -90.01deg", expectedResult: false, message: "'oblique' followed by positive out of range angle is in invalid" },
```

Comment:
Another duplicate test name:

```
`Font-style: 'oblique' followed by positive out of range angle is in invalid` | **PASS: 20/10**
```

Reply:
Ah, that's what 20/10 means. That wasn't clear to me. Hopefully fixed now?